app/enterprise/2.1.x/developer-portal/administration/application-registration/okta-config.md

```diff
@@ -98,6 +100,13 @@ your Okta application will vary:
 
 Now that the application has been configured in Okta, you need to associate the
 Okta application with the corresponding application in Kong's Developer Portal.
+
+<div class="alert alert-warning">
+  <strong>Note:</strong> Each developer should have their own application in both Okta and Kong.  
+  Each Okta application has its own `client_id` that maps to its respective developer in Kong.
```

Comment:
Should this say maps to its respective `application` in Kong?

Reply:
> Should this say maps to its respective `application` in Kong?

Yes, I had just used Satwant's verbiage verbatim https://kongstrong.slack.com/archives/CAW1MAFLZ/p1599077998045600.